NEWS

```diff
@@ -1,3 +1,52 @@
+KLEE 1.4.0, 20 July 2017
+========================
+(Incorporating changes from 4 November 2016 up to and including 20 July 2017)
+
```

Comment:
It might be useful to state what the corresponding release documentation commit is. I've just created the release 1.4.0 documentation.

https://github.com/klee/klee.github.io/pull/90